src/home/homework.c

```diff
@@ -1067,16 +1068,19 @@ static int home_deactivate(UserRecord *h, bool force) {
 }
 
 static int copy_skel(UserRecord *h, int root_fd, const char *skel) {
+        _cleanup_close_ int skel_fd = -EBADF;
         int r;
 
         assert(h);
         assert(root_fd >= 0);
 
-        r = copy_tree_at(AT_FDCWD, skel, root_fd, ".", h->uid, user_record_gid(h), COPY_MERGE|COPY_REPLACE, NULL, NULL);
+        r = chaseat(AT_FDCWD, skel, CHASE_MUST_BE_DIRECTORY, NULL, &skel_fd);
         if (r == -ENOENT) {
                 log_info("Skeleton directory %s missing, ignoring.", skel);
                 return 0;
         }
+
+        r = copy_tree_at(skel_fd, NULL, root_fd, ".", h->uid, user_record_gid(h), COPY_MERGE|COPY_REPLACE, /* denylist= */ NULL, /* subvolumes= */ NULL);
```

Comment:
and similar here, please add `/* xyz= */` comment to 2nd arg.

and maybe line break this, kinda long